ara/ui/templates/index.html

```diff
@@ -134,8 +134,20 @@
             </td>
             <td class="text-nowrap">{{ playbook.started | format_date }}</td>
             <td class="text-nowrap">{{ playbook.duration | format_duration }}</td>
-            <td class="text-center">{{ playbook.items.hosts }}</td>
-            <td class="text-center">{{ playbook.items.tasks }}</td>
+            <td class="text-center">
+              {% if not static_generation %}
+                <a href="{% url 'ui:host_index' %}?playbook={{ playbook.id }}" title="Search for hosts in this playbook">{{ playbook.items.hosts }}</a>
+              {% else %}
+                {{ playbook.items.hosts }}
+              {% endif %}
+            </td>
+            <td class="text-center">
+              {% if not static_generation %}
+              <a href="{% url 'ui:task_index' %}?playbook={{ playbook.id }}" title="Search for tasks in this playbook">{{ playbook.items.tasks }}</a>
```

Comment:
It's a different view: in the playbook report we actually have results (which are associated to tasks), not tasks themselves.

Reply:
yeah I know, that's why I was writen about "task results" and "tasks" ;)
I've added one more general comment regarding those linking things: https://github.com/ansible-community/ara/pull/342#issuecomment-1245175356

Reply:
I see what you mean.

The UI has grown organically, one page at a time, but it may eventually be time take a step back and consider the UX now that we have a lot more than just the playbook list and report page.